.github/workflows/nightly_tests_and_veracode.yml

```
@@ -74,14 +71,16 @@ jobs:
   nightly_tests:
     name: "Run nightly tests"
     runs-on: ubuntu-latest
-    if: github.event_name == 'schedule'
+    if: github.event_name == 'pull_request'
     steps:
       - name: "Checkout repository"
         uses: actions/checkout@v4
       - name: "Set up JDK"
         uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v8.14.1
+      - name: "Build SNAPSHOT dependencies"
+        run: mvn ${{ env.MAVEN_CLI_OPTS }} clean install -DskipTests -pl common-authentication,common-test
       - name: "Build application"
-        run: mvn clean install -DskipTests -pl live-ingester
+        run: mvn ${{ env.MAVEN_CLI_OPTS }} clean install -DskipTests

```

Oh. Ok, well this builds everything, making it pointless to do the previous step. I think we should get rid of "Build SNAPSHOT dependencies" and just do:
```suggestion
        run: mvn ${{ env.MAVEN_CLI_OPTS }} clean install -DskipTests -pl live-ingester -am
```

---

thanks, change is implemented